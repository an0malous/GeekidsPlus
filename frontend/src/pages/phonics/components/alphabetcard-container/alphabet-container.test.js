import React from 'react';
import renderer from 'react-test-renderer';
import AlphabetContainer from './alphabet-container.component';

describe('Alphabet Card Container', () => {
	let props;
	let component;

	beforeEach(() => {
		props = {
            currentWords: [ {name: "push", type: "blends", audio: null, img: null}],
				currentDeckIndex: 0
			}
				const {currentDeckIndex, currentWords } = props
		component = renderer
		.create(<AlphabetContainer currentDeckIndex={currentDeckIndex} currentWords={currentWords}/>)
		.toJSON()
	});

	it('renders without crashing', () => {
		expect(component).toMatchSnapshot();
   });

})


