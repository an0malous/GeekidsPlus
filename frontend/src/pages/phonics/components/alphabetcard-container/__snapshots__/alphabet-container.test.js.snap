// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alphabet Card Container renders without crashing 1`] = `
<div
  className="ui grid"
>
  <div
    className="centered row"
  >
    Loading Alphabet Cards...
  </div>
</div>
`;
